mypy/nodes.py

```diff
@@ -2184,6 +2185,7 @@ def __init__(self, names: 'SymbolTable', defn: ClassDef, module_name: str) -> No
         self.assuming_proper = []
         self.inferring = []
         self.add_type_vars()
+        self.replaced = TYPEINFO_NO_REPLACED
```

Comment:
I would actually make this `Optional`, it is used in few places, so should be easy to fix. Also I think we have an issue about removing `replaced` altogether.

Reply:
Oh, that was easy.